### ISSUE TITLE
expect-expect - false warning

### ISSUE THREAD

**Issue by EvgenyOrekhov** [playwright-community/eslint-plugin-playwright#149]
This gives false `expect-expect` warning

```ts
test('2 tabs', async ({ context, page }) => {
  await test.step('first tab', async () => { // <-- Test has no assertions (false warning)
    const frame = getFrame(page);
    await expect(frame.getByText('Hello')).toBeVisible();
  });

  await test.step('second tab', async () => {
    const frame2 = await openNewTabAndGetFrame(context);
    await expect(frame2.getByText('Hello')).toBeVisible();
  });
});
```

**Comment by mskelton** [comment]
Thanks for reporting, I've pushed a fix for this.

**Comment by AntonNiklasson** [comment]
Any chance we could get a release out for this @mskelton? 👏
